lib/snyk/enrich_cyclonedx.go

```diff
@@ -204,10 +204,21 @@ func enrichCycloneDX(bom *cdx.BOM, logger *zerolog.Logger) *cdx.BOM {
 
 				if issue.Attributes.Severities != nil {
 					for _, sev := range *issue.Attributes.Severities {
-						source := cdx.Source{
-							Name: "Snyk",
-							URL:  "https://security.snyk.io",
+						var source cdx.Source
```

Comment:
Is there something similar happening in SPDX land where we currently default to Snyk?

Reply:
In SPDX we enrich components with the snyk vulnerability db url as an external reference, but that's not exactly the same thing as it's not in the context of ratings, so imho we shouldn't touch the SPDX part.